db/migrate/20240408101038_add_timestamps_to_users_roles_table.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimestampsToUsersRolesTable < ActiveRecord::Migration[7.0]
+  def change
+    add_timestamps :users_roles, default: -> { 'CURRENT_TIMESTAMP' }
```

Comment:
I have a slight reservation because I've never seen it done this way before. It usually doesn't have the `default` area and is instead accompanied by a script. That being said, this seems to be working as expected.